src/main/java/cloud/fogbow/ras/core/OrderController.java

```diff
@@ -119,13 +119,62 @@ public void closeOrder(Order order) throws InternalServerErrorException {
 
     public void pauseCompute(Order order) throws FogbowException {
         synchronized (order) {
+            OrderState orderState = order.getOrderState();
+            if (order.isRequesterLocal(this.localProviderId) && hasOrderDependencies(order.getId())) {
```

Comment:
Why should an exception be raised if the order has dependencies? In the case of a Compute order, dependencies may come from a PublicIp, a Network or an Attachment orders. The only one that could lead to a special treatment is the attachment. I would remove the second part of the test and see what happen when a compute that has an attachment associated is paused. We can then discuss what to do, once we know Openstack's behavior.